app/controllers/admin/items_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::ItemsController < Admin::BaseController
+  def index
+    @items = Item.where(user_id: params[:merchant_id])
+    render :'merchant/items/index'
```

Comment:
great workaround

Reply:
I agree, way to reuse code with the render!